packages/ebayui-core-react/.storybook/preview.jsx

```diff
@@ -10,7 +10,7 @@ export default {
     decorators: [
         (Story) => (
             <StrictMode>
-                <EbaySvg />
+                <EbaySvg flags />
```

Comment:
Maybe it might be better to have this as a separate component, such as `EbayFlagsSvg`. It seems small enough.

Reply:
Yep yep! Let me do that! My initial idea was that the current `<ebay-flag>` was easier to do it inline as a span in the phone input and use it instead for the SVG since we do have the same for icon. But since the plan is to replace it with sprite, I will update as you mentioned.

Reply:
Changed to `EbayFlagsSvg` and we now have `EbayFlag` and `EbayFlagIcon`